<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_sha256">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Hashage - SHA256</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rsa">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Chiffrement - RSA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Chiffrement - AES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chiffreRSA">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Chiffrer avec clef public existante</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dechiffreRSA">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Dechiffrer avec clef privé existante</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dechiffreAES">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Dechiffrer avec clef secrète existante</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chiffreAES">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Chiffrer  avec clef secrète  existante</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generationclef">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Generation clefs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keyrsa">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>RSA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keyaes">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(32, 91, 83);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>981</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;

background-image: url(:/mapload.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>btn_sha256</zorder>
   <zorder>btn_rsa</zorder>
   <zorder>btn_aes</zorder>
   <zorder>chiffreRSA</zorder>
   <zorder>dechiffreRSA</zorder>
   <zorder>dechiffreAES</zorder>
   <zorder>chiffreAES</zorder>
   <zorder>generationclef</zorder>
   <zorder>keyrsa</zorder>
   <zorder>keyaes</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
